modules/channel-web/src/views/lite/components/Composer.tsx

```diff
@@ -23,6 +23,12 @@ class Composer extends React.Component<ComposerProps> {
   }
 
   handleKeyPress = async e => {
+    if (e.ctrlKey && e.key === 'Enter') {
+      await this.props.resetSession()
+      await this.props.sendMessage()
+      e.preventDefault()
+      return
+    }
```

Comment:
In most messaging service, CTRL+Enter adds a new line. I think that most users would be puzzled if their convo just disappeared. Please add a configuration option, which would be enabled specifically in the studio (in full/index.jsx). Something like enableNewSessionShortcut.

Otherwise, nice addition !